packages/xo-web/src/common/form/index.js

```diff
@@ -57,7 +62,12 @@ export class Password extends Component {
   }
 
   render() {
-    const { className, enableGenerator = false, ...props } = this.props
+    const {
+      className,
+      defaultVisible,
```

Comment:
It's not used.

Reply:
But it needs to be removed from the `props` before they're passed to the `input`.